frontend/projects/upgrade/src/app/features/dashboard/feature-flags/pages/feature-flag-root-page/projected-components/feature-flag-root-section-card-list/projected-components/feature-flag-root-table-section-card/projected-components/feature-flag-root-table-section-card-header-container/feature-flag-root-table-section-card-header-container.component.scss

```diff
@@ -0,0 +1,9 @@
+.section-card-header-container {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
```

Comment:
do we need align item here? already using padding so

Reply:
The section-card stuff is sort of an incomplete draft. Whoever takes the section card stuff can feel free to completely change the quick css I put in to demo the general idea, I think I told @zackcl  that I'd leave it in rather than remove it for this PR.